Sources/Attribution/AfficheClientProxy.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  AfficheClientProxy.swift
+//
+//  Created by Juanpe Catalán on 14/7/21.
+//
+
+import Foundation
+
+typealias AttributionDetailsBlock = ([String: NSObject]?, Error?) -> Void
```

Comment:
I would move this inside `AfficheClientProxy` to namespace it instead of putting it in the global namespace.
Also could this be `Result<[String: NSObject], Error>`?